beta-src/src/utils/drawArrow.ts

```diff
@@ -0,0 +1,49 @@
+import ArrowColors from "../enums/ArrowColors";
+
+export default function drawArrow(
+  /**
+   * color code passed from enum ArrowColors based on move type
+   */
+  arrowColor: ArrowColors,
+  /**
+   * source is the id of the element the arrow begins at
+   */
+  source: string,
+  /**
+   * svg parent element; must be able to target centers and defs
+   */
+  svgElement: any,
```

Comment:
any could just be anything. i could pass <div>anything</div> and it would not give me a typescript error, but this code would fail. i think you want SVGSVGElement